projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -205,8 +218,12 @@ jQuery( document ).ready( function ( $ ) {
 					$( '#jitm-banner__activate a' ).attr( 'disabled', true );
 				},
 			} ).done( function () {
-				$( '#jitm-banner__activate a' ).text( window.jitm_config.activated_module_text );
-				$( '#jitm-banner__activate a' ).attr( 'disabled', true );
```

Comment:
I think we'd want to keep these lines in the case that the activation button did not have a settings button to show. Could these lines just be moved to after the new if statement you added?